### PR TITLE
feat: translated many things

### PR DESCRIPTION
**Pull request by DisembodiedHead**
all items, haldor speech, most ravens & hildir speech, and many other things; some things were written in georgian, some had atrocious spelling - this has been fixed accordingly

### CODE REVIEW

**Review by DisembodiedHead — COMMENTED** [review]
you might notice that there's some places where the word "long" has been replaced with "Дълга" - fear not. this is a minor setback and i will be easily fixing this in future commits as i am myself a native bulgarian

oh i forgot to translate Black metal missile at line 11. this is unfortunate.

File: JotunnLib/Documentation/data/localization/translations/Bulgarian.md [hunk 1]
```diff
@@ -1608,273 +1608,273 @@ This file is automatically generated from Valheim 0.217.14 using the JotunnDoc m
...
-|$menu_players|Player list|
-|$menu_pleasewait|Please wait|
-|$menu_portals|Portals|
-|$menu_portals_tooltip|Changes how portals work in the game.|
-|$menu_presets|Presets|
-|$menu_privilegerequiredheader|Platform privilege required|
-|$menu_public|Публичен|
-|$menu_recent|Recent|
-|$menu_reloadfailed|Failed to reload saves|
+|$menu_passwordКъса|Паролaта е твърде кратка|
```
Comment (DisembodiedHead) [inline JotunnLib/Documentation/data/localization/translations/Bulgarian.md:1799]: oh my god.

File: JotunnLib/Documentation/data/localization/translations/Bulgarian.md [hunk 2]
```diff
@@ -1608,273 +1608,273 @@ This file is automatically generated from Valheim 0.217.14 using the JotunnDoc m
...
+|$menu_modifier_popup_text|Мола отбележете, че тези настройки могат да нарушат вашето преживяване във Валхайм. За първата ви игра е силно препоръчително да се играе с оригиналните настройки, като така ще можете да играете както е било измислено да се играе.|
+|$menu_modifier_popup_title|Внимание: модификации на свят|
+|$menu_modifier_veryeasy|Много лесно|
+|$menu_modifier_veryhard|Много трудно|
+|$menu_more|Още|
+|$menu_movefailedheader|Преместване на файла бе неуспешно!|
+|$menu_movetocloud|Премести в облак|
+|$menu_movetolocal|Премести в локален диск|
+|$menu_muchless|Още по-малко|
+|$menu_muchmore|Още повечр|
```
Comment (DisembodiedHead) [inline JotunnLib/Documentation/data/localization/translations/Bulgarian.md:1772]: YOU.

**Comment by MSchmoecker** [comment]
Unfortunately, these translations are only extracted from the game and are not directly affiliated with Valheim.

Please reach out to IronGate, the studio behind Valheim, at the [Valheim Discord](https://discord.com/invite/valheim) or via E-mail at [contact@irongatestudio.se](mailto:contact@irongatestudio.se)

**Comment by DisembodiedHead** [comment]
ah, my mistake then - thank you

**Pull request closed without merging**